src/esp/physics/bullet/BulletRigidObject.h

```diff
@@ -46,13 +46,13 @@ struct SimulationContactResultCallback
    * @param cp Contains detailed information about the contact point being
    * added.
    */
-  btScalar addSingleResult(btManifoldPoint& cp,
-                           const btCollisionObjectWrapper* colObj0Wrap,
-                           int partId0,
-                           int index0,
-                           const btCollisionObjectWrapper* colObj1Wrap,
-                           int partId1,
-                           int index1) {
+  btScalar addSingleResult(btManifoldPoint&,
+                           const btCollisionObjectWrapper*,
+                           int,
+                           int,
+                           const btCollisionObjectWrapper*,
+                           int,
+                           int) {
```

Comment:
You can use `CORRADE_UNUSED` to keep the variable names for semantic understanding while suppressing warnings.